webpack.config.client.js

```diff
@@ -1,6 +1,13 @@
 /* eslint-disable global-require */
 const AssetsPlugin = require('assets-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
+const dotenv = require('dotenv');
+
+const result = dotenv.config();
+
+if (result.error) {
+  throw result.error;
+}
```

Comment:
This is, somehow, causing files to be requested from `http://localhost:7080/news/articles/undefined/static/js/vendor-678f84af.a7416d5c.js` locally on build & start